keps/sig-release/3000-artifact-distribution/README.md

```diff
@@ -0,0 +1,135 @@
+
+### Non-Goals
+
+Anything related to creation of artifacts, bom, digital signatures.
+
+## Proposal
+
+There are two intertwined concepts that are part of this proposal.
+
+First, the policy and procedures to promote/upload our artifacts to multiple providers. Our existing processes upload only to GCS buckets. Ideally we extend the existing software/promotion process to push directly to multiple providers. Alternatively we use a second process to synchronize artifacts from our existing production buckets to similar constructs at other providers.
+
+Additionally we require a registry and artifact url-redirection solution to the local cloud provider or country.
+
+### User Stories
+
+#### SIG Release - Image Promotion
+
+```feature
+As a SIG Release volunteer
+I want to promote our binaries/images to multiple clouds
+
+Given a promotion / manifest
+When my PR is merged
+Then the promotion process occurs
+```
+
+#### Cloud Customer - Installing K8s via kubeadm
+
+```feature
+As a CLOUD end-user
+I want to install Kubernetes
+
+Given some compute resources at CLOUD
+When I use kubeadm to deploy Kubernetes
+Then I will be redirected to a local CLOUD registry
+```
+
+### Notes/Constraints/Caveats
+
+The primary purpose of the KEP is getting consensus on the agreed policy and procedure to unblock our community and move forward together.
+
+There has been a lot of activity around the technology and tooling for both goals, but we need shared agreement on policy and procedure first.
+
+### Risks and Mitigations
+
+This is the primary pipeline for delivering Kubernetes worldwide. Ensuring the appropriate SLAs and support as well as artifact integrity is crucial.
+
+## Design Details
+
+### Artifact Promotion
+
+#### Policy
+
+(more details needed, #sig-release-eng?)
+
+#### Process
+
+Currently the promotion process is primarily driven by the CIP/[promo-tool#kpromo](https://github.com/kubernetes-sigs/promo-tools#kpromo)?
+
+### Artifact Distribution
+
+#### Policy
+
+#### Process
+
+## Alternatives / Background
+
+- Apache has a widespread mirror network
+  - @dims has experience here
+  - http://ws.apache.org/mirrors.cgi
+  - https://infra.apache.org/mirrors.html
+- [Umbrella issue: k8s.gcr.io => registry.k8s.io solution k/k8s.io#1834
+](https://github.com/kubernetes/k8s.io/issues/1834)
+- [ii/registry.k8s.io Implementation proposals](https://github.com/ii/registry.k8s.io#registryk8sio)
+- [ii.nz/blog :: Building a data pipline for displaying Kubernetes public artifact traffic
+](https://ii.nz/post/building-a-data-pipline-for-displaying-kubernetes-public-artifact-traffic/)
+
+### How much is this going to save us?
+
+![Cost of K8s Artifact hosting - Data Studio Graphs](https://i.imgur.com/LAn4UIE.png)
+
+## Infrastructure Needed
+
+It would be good to request some donations for some larger providers, including one in China, via [Cloud Native Credits program](https://www.cncf.io/credits/).
```

Comment:
We should provide a list of what exactly is needed in terms of infrastructure to ensure this KEP go to `implementable`.